### PR TITLE
Update StatusBar.java

### PR DESCRIPTION
**Pull request by oarsheo**
`Read 'StatusBarOverlaysWebView' from config.xml, default is false.`

`setStatusBarTransparent(preferences.getBoolean("StatusBarOverlaysWebView", false));`

### CODE REVIEW

**Comment by janpio** [comment]
Can you please provide some context for this change?
Why is this needed? What problem does it solve?

**Comment by oarsheo** [comment]
Parameter StatusBarOverlaysWebView for android not working
This commit solves this problem

**Comment by breautek** [comment]
Just want to make a point that the docs currently states this defaults to true, but obviously doing so may break apps that depends on the android's broken result, ie no status bar overlay by default)

File: src/android/StatusBar.java [hunk 1]
```diff
@@ -105,6 +108,8 @@ public void run() {
                     // CB-11197 We still need to update LayoutParams to force status bar
                     // to be hidden when entering e.g. text fields
                     window.clearFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN);
+                    
+                    setStatusBarTransparent(true);
```
Comment (timbru31) [inline src/android/StatusBar.java]: Is this line related to the bug #151 or unrelated?

File: src/android/StatusBar.java [hunk 2]
```diff
@@ -64,6 +64,9 @@ public void run() {
 
                 // Read 'StatusBarStyle' from config.xml, default is 'lightcontent'.
                 setStatusBarStyle(preferences.getString("StatusBarStyle", "lightcontent"));
+
+                // Read 'StatusBarOverlaysWebView' from config.xml, default is false.
+                setStatusBarTransparent(preferences.getBoolean("StatusBarOverlaysWebView", false));
```
Comment (timbru31) [inline src/android/StatusBar.java]: While the default is broken, I'd rather fix the bug in the way that it corresponds with the docs, i.e., true.

Comment (Menardi) [inline src/android/StatusBar.java]: `setStatusBarTransparent` should be called before setting the background colour, or else the background colour will be ignored and the statusbar will always be fully transparent.

Comment (breautek) [reply src/android/StatusBar.java]: I've raised an issue describing this at https://github.com/apache/cordova-plugin-statusbar/issues/159 because I think the statusbar plugin should be smarter in handling this. But for now, I agree, the workaround is to invoke `setStatusBarTransparent` before `setStatusBarBackgroundColor`.

**Comment by mosababubakr** [comment]
When this PR will be done ?

**Comment by timbru31** [comment]
Closing due to inactivity and we've merged in #171 which correctly reads the variable as true, like it's stated.
